base_layer/core/src/base_node/service/service.rs

```diff
@@ -297,8 +297,14 @@ where B: BlockchainBackend + 'static
         task::spawn(async move {
             let result = handle_incoming_block(inbound_nch, new_block).await;
 
-            if let Err(e) = result {
-                error!(target: LOG_TARGET, "Failed to handle incoming block message: {:?}", e);
+            match result {
+                Err(BaseNodeServiceError::CommsInterfaceError(CommsInterfaceError::ChainStorageError(
+                    ChainStorageError::CannotAcquireFileLock,
```

Comment:
This error only happens when you startup a node with the same base dir - the db lockfile is present. 
`Cannot acquire exclusive file lock, another instance of the application is already running`

Which error message are you trying to suppress?

Reply:
good catch, wrong error.